vue-components/src/components/Dropdown.vue

```diff
@@ -227,6 +227,7 @@ $base: '.wikit-Dropdown';
 			border-color: $wikit-Dropdown-active-border-color;
 			background-color: $wikit-Dropdown-active-background-color;
 			box-shadow: $wikit-Dropdown-active-box-shadow;
+			outline: none;
```

Comment:
We should probably comment inline that this affects only Safari's behavior.
I can see how someone can remove it when they see it doesn't do anything on Chrome/Firefox.